<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Process</class>
 <widget class="QMainWindow" name="Process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sizeLabel">
        <property name="text">
         <string>内存大小：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sizeInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="sizeButton">
        <property name="text">
         <string>设置内存大小</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="valueInputLabel">
      <property name="text">
       <string>传输的值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="valueInput"/>
    </item>
    <item>
     <widget class="QLabel" name="valueOutputLabel">
      <property name="text">
       <string>读取的值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="valueOutput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="writeButton">
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readButton">
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="memoryBlockTable">
      <column>
       <property name="text">
        <string>地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>大小</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="logTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="memoryUsageLabel">
        <property name="text">
         <string>内存使用率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="memoryUsageBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Process.qrc"/>
 </resources>
 <connections/>
</ui>
